### ISSUE TITLE
About typos, small grammar fixes, etc.

### ISSUE THREAD

**Issue by cadadr** [djnavarro/rbook#31]
Hello,

This is a great book and I'm loving it as a more pragmatic follow up to the rather theoretical but still nice Statistics in Plain English by Urdan (which was the book we used in our research methods course---FWIW, I'm doing an MA in linguistics), many thanks for authoring it and publishing it as an open-source resource.

Being the pedant that I am, and having access to sources, I'm fixing typos and some wording as I go along the book. I wanted to ask, if you'd want the fixes submitted as a single PR, or on a case by case basis. I've 30+ edits by now and most are just typos so wouldn't really warrant a review, but a couple are wording so you might want to reject some changes. I think a single PR would be better because so many are minor edits so maybe just picking hunks you like from a big commit is more efficient, but I can send them one by one in distincts PRs if you prefer it that way.

**Comment by djnavarro** [comment]
Perhaps one big PR is easiest? 

Also, thank you!

**Comment by cadadr** [comment]
You're welcome! Alright, I'll close this one and send a PR when I'm done reading. Stay safe!
